gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -39,6 +39,11 @@
         <property name="format" value="\s+$"/>
         <property name="message" value="Whitespace at end-of-line"/>
     </module>
+    <module name="RegexpMultiline"> <!-- no double line breaks -->
+        <property name="fileExtensions" value="java"/>
+        <property name="format" value="^\n\n$"/>
+        <property name="message" value="Multi-line breaks"/>
```

Comment:
let's make this a littler friendlier: "Two consecutive blank lines are not permitted"